src/Compilers/Core/Portable/Compilation/Compilation.cs

```diff
@@ -1527,6 +1527,58 @@ internal bool IsRealSigned
             Predicate<ISymbol> filterOpt,
             CancellationToken cancellationToken);
 
+        internal bool StartSourceChecksumCalculation(DebugDocumentsBuilder documentsBuilder, DiagnosticBag diagnostics)
```

Comment:
I have just moved the method from C# down to CodeAnalysis. You're right though, the name is odd. I'll rename it to CalculateSourceChecksums.